src/main/webapp/assets/modules/pie-chart/pie-chart.fac.js

```diff
@@ -71,7 +71,10 @@ angular.module('Pie').factory('pieFactory', function (utils, jsonValue, termServ
           }
         },
         tooltip: {
-          pointFormat: '{series.name} <b>: {point.percentage:.1f}%</b>'
+          formatter: function() {
+            return sprintf("<b>%(salRange)s</b> <br/>a month in average for jobs in <b>%(label)s</b>",
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'sprintf' is not defined.